samples/matter/light_bulb/sample.yaml

```diff
@@ -39,12 +32,12 @@ tests:
       - nrf21540dk_nrf52840
     platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
   # ---------------
-  sample.matter.light_bulb.debug.nrf5340dk:
+  sample.matter.light_bulb.debug:
     build_only: true
     integration_platforms:
       - nrf5340dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
-  sample.matter.light_bulb.debug.nrf7002_ek:
```

Comment:
why debug is removed while the device is build as debug configuration

Reply:
if no `.debug` in sample name means that device is in debug configuration, then it need to be align across all sample configuration definitions

Reply:
I think with those changes it will be consistent across all samples.yaml, that "debug" in name will be only if there are no additional parameters changed from default set.